css/templates/lune/components/_chapter.css

```diff
@@ -11,7 +11,7 @@
 .lune-chapter-name,
 .lune-chapter-number {
   /* Width is one letter wide */
-  width: 1em;
+  width: .7em;
```

Comment:
Lowered the width slightly, as an "I" would stay on the same line as the next letter